lists-ui/src/views/Dashboard/components/Breadcrumbs.tsx

```diff
@@ -37,21 +48,28 @@ export function Breadcrumbs() {
           <Anchor href='https://ala.org.au' className={classes.link} size='sm'>
             Home
           </Anchor>
-          {isNested ? (
+          {parts.length > 0 ?  (
             <Anchor component={Link} to='/' className={classes.link} size='sm'>
               Species lists
             </Anchor>
           ) : (
             <Text size='sm'>Species lists</Text>
           )}
-          {isNested && (
+          {/* Link to the specific list (if applicable and not the last item) */}
+          {isListPage && isNestedAction && listTitle && (
```

Comment:
Should this be `||`? I am not familiar enough with routes in this application to be certain.

Reply:
Yeah, see what you mean. I'll test and fix if needed.